test/BuildingRegistry.Console.ReactivatedHouseNumbers/Program.cs

```diff
@@ -0,0 +1,104 @@
+namespace BuildingRegistry.Console.ReactivatedHouseNumbers
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using Amazon.DynamoDBv2;
+    using Api.CrabImport.CrabImport.Requests;
+    using Be.Vlaanderen.Basisregisters.Crab;
+    using Be.Vlaanderen.Basisregisters.EventHandling;
+    using Be.Vlaanderen.Basisregisters.GrAr.Import.Processing.Json;
+    using Building.Commands.Crab;
+    using Building.Events.Crab;
+    using Dapper;
+    using Microsoft.Data.SqlClient;
+    using Microsoft.Extensions.Configuration;
+    using Newtonsoft.Json;
+    using NodaTime;
+
+
+    //Case 5f: when housenumber relation is retired then becomes active again (correction), buildings with more than 2 subaddresses don't have the right amount of units
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: false)
+                .AddJsonFile($"appsettings.{Environment.MachineName.ToLowerInvariant()}.json", optional: true, reloadOnChange: false)
+                .AddEnvironmentVariables()
+                .AddCommandLine(args ?? new string[0])
+                .Build();
+
+            var eventsConnectionString = configuration.GetConnectionString("Events");
+            var jsonSerializerSettings = new JsonSerializerSettings().ConfigureDefaultForEvents();
+            int count = 0;
+            var lastStreamId = -1;
+
+            // GET STREAMS
+            using (var sqlConnection = new SqlConnection(eventsConnectionString))
+            {
+                var streamsIds = sqlConnection.Query<int>(
+                    @"SELECT TOP(10000) IdInternal FROM [building-registry-events].[BuildingRegistry].[Streams] WHERE IdInternal > @lastStreamId ORDER BY IdInternal", new { lastStreamId })
```

Comment:
is it an option to join this with messages right away so you can filter out streams which have no `TerrainObjectHouseNumberWasImportedFromCrab` anyway? (unless they usually all have it)